.gitignore

```diff
@@ -1,35 +0,0 @@
-# General files for the project
```

Comment:
You need to remove this file from your commit.  You are removing it from the repo.  Add it back in your branch please.